src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -198,7 +210,7 @@ final class OnboardingInfoViewController: UIViewController {
 	// MARK: Exposure notifications
 
 	private func askExposureNotificationsPermissions(completion: (() -> Void)?) {
-		if TestEnvironment.shared.isUITesting {
+		guard !TestEnvironment.shared.isUITesting && !exposureManagerActivated else {
```

Comment:
I don't like `TestEnvironment.shared.isUITesting` inside our code but that's not part of this PR

Reply:
Do we have a preferred way to check for UI testing state at the moment?